WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListActivity.java

```diff
@@ -106,14 +122,96 @@ public void onBackPressed() {
     }
 
     @Override
-    public boolean onOptionsItemSelected(final MenuItem item) {
+    public boolean onCreateOptionsMenu(Menu menu) {
+        // add a follow menu item to the toolbar for tag/blog preview
+        if (getPostListType().isPreviewType()) {
+            mFollowMenuItem = menu.add(R.string.reader_btn_follow);
+            mFollowMenuItem.setShowAsAction(MenuItem.SHOW_AS_ACTION_ALWAYS);
+            updateFollowMenu();
+        }
+        return super.onCreateOptionsMenu(menu);
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
         if (item.getItemId() == android.R.id.home) {
             onBackPressed();
             return true;
+        } else if (mFollowMenuItem != null && item.equals(mFollowMenuItem)) {
```

Comment:
nitpick; no need to check if mFollowMenuItem here, `.equals` should take care of that.